chatbot_ner/config.py

```diff
@@ -39,6 +40,35 @@
 ner_logger.addHandler(handler)
 ner_logger.addHandler(handler_stdout)
 
+# HAPTIK Environment and CAS name
+CLIENT_APPLICATIONS_SETUP_NAME = os.environ.get('CLIENT_APPLICATIONS_SETUP_NAME')
+HAPTIK_ENV = os.environ.get('HAPTIK_ENV')
+
+# Support for Sentry DSN
+SENTRY_DSN = os.environ.get('SENTRY_DSN')
+SENTRY_ENABLED = os.environ.get('SENTRY_ENABLED')
+SENTRY_ENABLED = True if SENTRY_ENABLED == 'True' and 'test' not in sys.argv else False
+SENTRY_ALLOWED_LOGGERS = os.environ.get('SENTRY_ALLOWED_LOGGERS')
+SENTRY_ALLOWED_LOGGERS = [sentry_allowed_logger.strip() for sentry_allowed_logger
+                          in SENTRY_ALLOWED_LOGGERS.split(',')] if SENTRY_ALLOWED_LOGGERS else []
+
+if SENTRY_ENABLED:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    from sentry_sdk.integrations.logging import LoggingIntegration
+
+    def before_sentry_send(event, hint):
+        event.setdefault("tags", {})["cas_name"] = CLIENT_APPLICATIONS_SETUP_NAME
+        return event
+
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        integrations=[DjangoIntegration(), LoggingIntegration()],
+        environment=HAPTIK_ENV,
+        sample_rate=0.1,
+        before_send=before_sentry_send
+    )
```

Comment:
should probably move this under a function to `chatbot_ner/setup_sentry.py`
and call that method in `chatbot_ner/settings.py`
This file is already very messy